packages/schema/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "author": "Google Labs Team",
+  "devDependencies": {
+    "@google-labs/tsconfig": "^0.0.1",
+    "@types/node": "^20.11.10",
+    "ajv": "^8.12.0",
+    "ajv-formats": "^2.1.1",
+    "wireit": "^0.14.4"
```

Comment:
I think we can just rely on the top-level wireit dependency. Otherwise when we bump versions we'll get duplicates. Same for any other shared deps (e.g. types/node?)

Reply:
thoughts on using something like syncpack?